pkg/client_data/lib/package_api.dart

```diff
@@ -154,11 +154,15 @@ class VersionInfo {
 class VersionScore {
   final int grantedPoints;
   final int maxPoints;
+  final int likeCount;
+  final double popularityScore;
```

Comment:
Can we rename this `popularity`

Reply:
I was pondering on this, and thought that if we'd call it `popularity`, then we should also transform it into the `0-100` range, and switch the type to `int`. wdyt?

Reply:
I'm also okay keeping it this way. let's not over think it, long term I think we want absolute numbers here.